Library/Homebrew/formula.rb

```diff
@@ -1364,6 +1364,18 @@ def self.clear_installed_formulae_cache
     @installed = nil
   end
 
+  def self.comparison
```

Comment:
Whoops, sorry about this misunderstanding. Just pushed out another change to move it there as a global method.